iterative/utils/logger.go

```diff
@@ -0,0 +1,104 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/sirupsen/logrus"
+	"strings"
+	"time"
+)
+
+var baseTimestamp = time.Now()
+var colors = make(map[string]int)
+
+type basicFormatter struct{}
+
+func (f *basicFormatter) Format(entry *logrus.Entry) ([]byte, error) {
+	levelText := strings.ToUpper(entry.Level.String())
+	levelColor := colors[levelText]
+	tpl := "[%s] 🚀\x1b[%dmTPI\x1b[0m %s\n"
```

Comment:
:nail_care: would prefer

```suggestion
	tpl := "%s:🚀\x1b[%dmTPI\x1b[0m:%s\n"
```

since it leaves more room for the message.

Reply:
Let me give it a shot

Reply:
It needs the spaces for readability and to align tf logs with logrus

Reply:
Same as https://github.com/iterative/terraform-provider-iterative/pull/423#discussion_r819149028 it seems.